From c735526d3060532ec885e4f959c4ccfaf4a86c30 Mon Sep 17 00:00:00 2001
From: auto commit <unknown@unknown>
Date: Fri, 24 Oct 2008 12:23:16 -0700
Subject: [PATCH] cavium: Kconfig for no FPU EMU stats.

Add in a Kconfig so that the collection of FPU EMU stats
can be either enabled or disabled without changing code.

Signed-off-by: Tomaso Paoletti <tpaoletti@caviumnetworks.com>
Signed-off-by: Paul Gortmaker <Paul.Gortmaker@windriver.com>
---
 arch/mips/Kconfig |    9 +++++++++
 1 files changed, 9 insertions(+), 0 deletions(-)

diff --git a/arch/mips/Kconfig b/arch/mips/Kconfig
index a3e90bc..cdcf2fd 100644
--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -1942,6 +1942,15 @@ config SECCOMP
 
 	  If unsure, say Y. Only embedded should say N here.
 
+config MIPS_FPU_EMULATOR_STATS_ENABLED
+	bool "Enable FPU emulator statics"
+	depends on SMP
+	default n
+	help
+	  On SMP systems the FPU emulator statistics cause huge amounts of cross CPU
+	  memory traffic that isn't really needed. Nobody actually has any way of
+	  reading these statistics.
+
 endmenu
 
 config RWSEM_GENERIC_SPINLOCK
-- 
1.5.5.1

